src/components/BasketballCourt/Border.tsx

```diff
@@ -9,18 +10,21 @@ const Border: React.FC<BorderProps> = ({ startPoint }) => {
   const { courtAreaXLength, courtAreaYLength, borderLength } = useStoreSelector(
     (state) => state.courtSize
   );
+  const currentColorBoardColor = useStoreSelector((state) => state.courtColor.color);
```

Comment:
I think "currentColorBoardColor" is not a proper name for this value.

Reply:
Thanks, I changed the name to be “selectedColor".